mongo/datadog_checks/mongo/data/conf.yaml.example

```diff
@@ -13,7 +13,7 @@ init_config:
 #
 instances:
 
-    ## @param hosts - list of strings - optional
+    ## @param hosts - string or list of strings - optional
```

Comment:
This file is the sample configuration that comes with integrations. It is autogenerated from the spec